src/main/java/org/reaktivity/nukleus/maven/plugin/internal/ast/AstMemberNode.java

```diff
@@ -91,6 +94,11 @@ public String sizeName()
         return sizeName;
     }
 
+    public boolean hasDefault()
+    {
+        return hasDefault;
+    }
+
```

Comment:
This doesn't feel quite right, only because `defaultValue` already exists, so they seem to overlap in the concepts they describe.

Suggest using a sentinel value for `private static final NULL_DEFAULT = new Object();` instead, such that `defaultValue == null` continues to mean no default, whereas `defaultValue == NULL_DEFAULT` means there is a default value, and it is `null`.

Reply:
done